mmdet/datasets/builder.py

```diff
@@ -206,10 +204,15 @@ def build_dataloader(dataset,
     return data_loader
 
 
-def worker_init_fn(worker_id, num_workers, rank, seed):
+def worker_init_fn(worker_id, num_workers, rank):
+    # according to https://github.com/open-mmlab/mmdetection/issues/8110
+    # numpy may be set the same seed at each epoch.
+    # Thus, # we use the return torch.initail_seed here to make sure numpy
+    # is set a new seed in each epoch.
```

Comment:
add https://pytorch.org/docs/stable/data.html#randomness-in-multi-process-data-loading link